src/index.ts

```diff
@@ -0,0 +1,78 @@
+const inputElement = document.getElementById("commentInput");
+const formElement = document.getElementById("inputForm");
+const ulElement = document.getElementById("commentsList");
+
+//submit form
+if (formElement !== null) {
+    formElement?.addEventListener("submit", (event) => {
+        event.preventDefault();
+        if (inputElement !== null) {
+            const newMsg = (inputElement as HTMLInputElement).value;
+            if (newMsg === "") {
+                return;
+            }
+            // li father
+            const newLiElement = document.createElement("li");
+            // profile pic
+            const newImgElement = document.createElement("img");
+            newImgElement.className = "profilePicture";
+            newImgElement.src = "https://cdn.pixabay.com/photo/2021/05/20/11/58/beauty-6268460_1280.jpg";
+            newImgElement.alt = "profile Picture";
```

Comment:
Move these into re-usable methods
ie. createProfilePicture

which will create the element and add all the necessary children and properties to it